app-backend/app/src/main/scala/image/Routes.scala

```diff
@@ -44,9 +44,9 @@ trait ImageRoutes extends Authentication
   }
 
   def createImage: Route = authenticate { user =>
-    entity(as[Image.Create]) { newImage =>
-      onSuccess(Images.insertImage(newImage.toImage(user.id))) { image =>
-        complete(StatusCodes.Created, image)
+    entity(as[Image.Banded]) { newImage =>
+      onSuccess(Images.insertImage(newImage, user)) { image =>
+        complete(image)
```

Comment:
This may be an API Style decision, but I believe `complete(image)` will return a `200 OK`, whereas `complete(StatusCodes.Created, image)` returned a more RESTful `201 CREATED`.